cit-api/pipeline/migrations/0123_auto_20210303_2236.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.16 on 2021-03-03 22:36
+
+from django.db import migrations, models
+
+
+def populate_statuses(apps, schema_editor):
+    ApprovalStatus = apps.get_model("pipeline", "ApprovalStatus")
+    statuses = [
+        ["Pending Edit from Community User/EDO", "Opportunity has been saved by submitter in an incomplete state.", "EDIT"],
```

Comment:
There's another migration that adds a status with the code EDIT already, does this one conflict with that one? I think the status code column has a unique constraint, so that might cause issues for someone migrating from scratch.

Reply:
This will work.... however, this wrong at a business level. I need to add a NEW type, and remove the EDIT type, ad EDIT and NCOM are the same thing in this adjust workflow.